examples/build.gradle

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,

```

Comment:
```suggestion
 * WSO2 LLC. licenses this file to you under the Apache License,

```

Reply:
Fixed in [ad28983](https://github.com/ballerina-platform/module-ballerinax-ai.weaviate/pull/8/commits/ad2898359536f755c33720488b968161ab29f959)